src/vm-memory/src/mmap.rs

```diff
@@ -380,6 +455,11 @@ impl GuestMemoryMmap {
 
         Err(Error::InvalidGuestRegion)
     }
+
+    /// Return true if dirty page tracking is enabled for `GuestMemoryMmap`, and else otherwise.
+    pub fn is_dirty_tracking_enabled(&self) -> bool {
+        self.regions[0].dirty_bitmap().is_some()
```

Comment:
In theory we might have a GuestMemory with no regions. Anyway I would use `self.regions.iter().all()` here.

Reply:
Yes, that's better.